jsonnet/components/thanos-querier.libsonnet

```diff
@@ -153,6 +139,33 @@ function(params)
       },
     },
 
+    // This is the KubeRBACProxy configuration for the web endpoint.
+    // In previous version, anyone with a "get" access on "namespace" resource
+    // can access the web endpoint. But KubeRBACProxy takes "get" verb as HTTP GET
+    // method, while the console access the web endpoint using HTTP POST method.
+    // A dedicated resource will be used to implementing this security setting.
```

Comment:
the comment would be more applicable to the cluster-monitoring-view cluster role.

Reply:
will move the comment to the cluster-monitoring-view cluster role.